Toolchain/BuildIt.sh

```diff
@@ -80,12 +80,6 @@ BINUTILS_NAME="binutils-$BINUTILS_VERSION"
 BINUTILS_PKG="${BINUTILS_NAME}.tar.gz"
 BINUTILS_BASE_URL="https://ftp.gnu.org/gnu/binutils"
 
-GDB_VERSION="13.1"
-GDB_MD5SUM="92b70971e81a450f6b3e1cf568671cfa"
```

Comment:
the removed md5sum doesn't match the md5sum in your new file? On a scale of "it looks right" to "ran it a few times", how tested is the new script? :D

Reply:
I snuck in a `.tar.gz` => `.tar.xz` change, that's why the hashes differ. It's been tested on Arch and arm64 macOS.

Reply:
Another possibly suspicious change is the removal of the `brew --prefix isl` lookup. Apparently, libisl is only used for an obscure, optional GCC feature (GRAPHITE). We aren't even building GCC in-tree, so it was completely unnecessary. I removed it in the latest force-push.